devops/continuous_integration/packages_pipeline/single_configuration.rst

```diff
@@ -0,0 +1,98 @@
+Package pipeline: single configuration
+======================================
+
+We will start with the most simple case, in which we only had to build 1 configuration, and that configuration 
+can be built in the current CI machine.
+
+As we described before while presenting the different server binary repositories, the idea is that package builds
+will use by default the ``develop`` repo only, which is considered the stable one for developer and CI jobs.
+
+Let's make sure we start from a clean state:
+
+.. code-block:: bash
+
+    $ conan remove "*" -c  # Make sure no packages from last run
+    $ conan remote remove "*"  # Make sure no other remotes defined
+    $ conan remote add develop <url-develop-repo>  # Add only the develop repo
```

Comment:
I think that most users will run all these commands for a local artifactory, so it would be great to offer them the full line for copying and just add a comment to change this if if differs

```suggestion
    # adjust these lines for your artifactory instance
    $ conan remote add develop http://localhost:8081/artifactory/api/conan/develop
    $ conan remote login develop admin -p password
```